.github/workflows/pytest-gpu.yaml

```diff
@@ -52,7 +52,6 @@ jobs:
       run: |
         set -ex
         python -m pip install mosaicml-cli
-        mcli init --mcloud
```

Comment:
was this just not necessary?

Reply:
yep - you just need an api key in mcli 0.5